src/reflect/scala/reflect/internal/util/ChromeTrace.scala

```diff
@@ -20,6 +20,7 @@ import scala.collection.mutable
 
 object ChromeTrace {
 
+  @annotation.unused // spurious
```

Comment:
This is because all the fields are inlined final val.

Reply:
Oh, thanks for the comment. I'm working on unused unuseds, and this is unused. Hmm. 😖